<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>José Eduardo de Souza Pimentel</author>
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prescrição</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pena máxima:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Data do fato:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Receb. denúncia:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditFato">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_2RecDen">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_3InicioSusp">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_4FimSusp">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxInicSusp">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Início da suspensão:</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2FimSusp">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fim da suspensão:</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCalc">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>210</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2Sair">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Promotoria de Justiça de Piracicaba</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxMetade">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Redução pela metade</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxPena">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>162</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContents</enum>
    </property>
    <item>
     <property name="text">
      <string>Pena  &lt;  1 ano</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 ano  &lt;= Pena &lt;= 2 anos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 anos  &lt;  Pena &lt;= 4 anos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4 anos  &lt;  Pena &lt;= 8 anos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8 anos  &lt;  Pena &lt;= 12 anos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pena  &gt; 12 anos</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBoxPena</tabstop>
  <tabstop>dateEditFato</tabstop>
  <tabstop>dateEdit_2RecDen</tabstop>
  <tabstop>checkBoxInicSusp</tabstop>
  <tabstop>dateEdit_3InicioSusp</tabstop>
  <tabstop>checkBox_2FimSusp</tabstop>
  <tabstop>dateEdit_4FimSusp</tabstop>
  <tabstop>checkBoxMetade</tabstop>
  <tabstop>pushButtonCalc</tabstop>
  <tabstop>pushButton_2Sair</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
